src/utils/translations.ts

```diff
@@ -100,12 +100,13 @@ export function getTranslatedPage(
 	const matchedResult = globResults.find((md) =>
 		md.file.endsWith(`${lang}.md`),
 	);
-	const enResult = globResults.find((md) => md.file.split(".")[1] === "md");
 
 	const locales = globResults
-		.map((md) => md.file.split(".")[1])
+		.map((md) => getLanguageFromFilename(md.file))
 		.filter(isLanguageKey);
 
+	const enResult = globResults[locales.findIndex((lang) => lang === "en")];
```

Comment:
```suggestion
	const locales = globResults
		.map((md) => getLanguageFromFilename(md.file));

	const enResult = globResults[locales.findIndex((lang) => lang === "en")];
```

Looking at this, it seems like the `.filter(isLanguageKey)` could throw off `.findIndex` if it filters anything. That said, since `getLanguageFromFilename` already returns a language, I don't think the filter is needed.